taskt/Core/Automation/Commands/IE Browser Commands/IECreateBrowserCommand.cs

```diff
@@ -65,39 +79,41 @@ public override void RunCommand(object sender)
             {
                 GlobalAppInstances.AddInstance(instanceName, newBrowserSession);
             }
-
-        }
-
-        public static void WaitForReadyState(SHDocVw.InternetExplorer ieInstance)
-        {
-            try
-            {
-                DateTime waitExpires = DateTime.Now.AddSeconds(15);
-
-                do
-                {
-                    System.Threading.Thread.Sleep(500);
-                }
-
-                while ((ieInstance.ReadyState != SHDocVw.tagREADYSTATE.READYSTATE_COMPLETE) && (waitExpires > DateTime.Now));
-            }
-            catch (Exception ex) { }
         }
 
         public override List<Control> Render(frmCommandEditor editor)
         {
             base.Render(editor);
 
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_InstanceName", this, editor));
+            RenderedControls.AddRange(CommandControls.CreateDefaultDropdownGroupFor("v_InstanceTracking", this, editor));
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_URL", this, editor));
 
             return RenderedControls;
         }
 
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + " [Instance Name: '" + v_InstanceName + "']";
+            return base.GetDisplayValue() + $" [Create IE Browser Instance '{v_InstanceName}']";
```

Comment:
This should be just [Instance Name '{v_InstanceName}']. Otherwise you're just repeating too much of the base.

Reply:
done